documentation/docusaurus.md

```diff
@@ -0,0 +1,76 @@
+# Scalar API Reference for Docusaurus
+
+Docusaurus helps you to ship a beautiful documentation site in no time. For everyone wo wants to make their API reference part of a Docusaurus website, we’ve built a Scalar API Reference plugin. Here is how you can integrate it in your project:
+
+## Create a new Docusaurus project (optional)
+
+If you’re starting ~~on a green field~~ fresh, let’s install Docusaurus first:
+
+> Note: It seems there are some issues with Docusaurus and npm. We’d recommend to [use pnpm](https://pnpm.io/installation), which is an awesome alternative to npm.
+
+```bash
+pnpm create create-docusaurus@latest my-awesome-website classic
+```
+
+If the installer asks you which language you prefer, pick whatever feels right. If you don’t know what TypeScript is, just use JavaScript. Both options will work great:
+
+```bash
+? Which language do you want to use?
+❯   JavaScript
+    TypeScript
+```
+
+You’ve got your project ready. Time to start the development server:
+
+```bash
+cd my-awesome-website
+pnpm start
+```
+
+Boom, that’s it. Open <http://localhost:3000/> to see the Docusaurus example documentation.
+
+## Render your OpenAPI reference with Scalar
+
+Okay, you’re ready to render your API reference with Docusaurus? Install our plugin:
+
+> Note: It seems there are some issues with Docusaurus and npm. We’d recommend to [use pnpm](https://pnpm.io/installation), which is an awesome alternative to npm.
+>
+> Otherwise you’ll probably receive something like:
+>
+> npm ERR! Cannot read properties of null (reading 'matches')
+
+```bash
+pnpm add @scalar/docusaurus
+```
+
+Done! There’s just one more step required: Adding the plugin to your Docusaurus configuration.
+
+```ts
+// docusaurus.config.js (or .ts if you’re project is in TypeScript)
+import { ScalarOptions } from '@scalar/docusaurus'
+
```

Comment:
should we describe both js/ts config examples? this would throw an error if not using ts so it might be misleading

Reply:
It had a comment helping with TypeScript/JavaScript, but I just added both examples. :)